src/features/wallet/send/SendLockedHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '../../../components/Box'
+import Close from '../../../assets/images/close.svg'
+import TouchableOpacityBox from '../../../components/TouchableOpacityBox'
+import Text from '../../../components/Text'
+
+type Props = {
+  onClosePress: () => void
+}
+
+const SendLockedHeader = ({ onClosePress }: Props) => {
```

Comment:
Could probably remove Send from this one too if we go with a generic approach

Reply:
sure I can make those more generic